lib/Partitioner/Partitioner.cpp

```diff
@@ -981,6 +1003,10 @@ void Partitioner::getBackendMap(
       // is the same.
       // TODO : will improve the algorithm for different memory size.
       backendInfo.memSize = deviceInfo_[i].availableMemory;
+      backendInfo.nonSupportedNodesKinds =
+          generateNodeKindsSet(deviceInfo_[i].nonSupportedNodes);
```

Comment:
Maybe `std::move(generateNodeKindsSet(deviceInfo_[i].nonSupportedNodes))` to avoid copying the set?